packages/ui/src/app/pages/Forum/ForumArchived.tsx

```diff
@@ -39,12 +39,7 @@ export const ForumArchived = () => {
         <>
           <RowGapBlock gap={24}>
             <ItemCount count={forumCategories?.length}>Archived Categories</ItemCount>
-            {isLoadingCategories ? (
-              <Loading />
-            ) : (
-              forumCategories &&
-              forumCategories.length > 0 && <ForumCategoryList categories={forumCategories} isArchive />
-            )}
+            {isLoadingCategories ? <Loading /> : forumCategories && forumCategories.length > 0 && <ForumMain />}
```

Comment:
The way I understand this should just be:
```suggestion
            <ForumMain isArchive />
```
But I'm confused that on the `joystream-testnet` on: https://dao-git-feature-forum-main-tab-joystream.vercel.app/#/forum/archived the threads and categories are empty instead of showing the same thing as: https://dao-git-feature-forum-main-tab-joystream.vercel.app/#/forum

Reply:
Forum archived is out of scope for this PR